Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionDeviceManagerProfileInspector.cs

```diff
@@ -79,9 +93,27 @@ public virtual void RenderCustomInspector()
                     {
                         EditorGUILayout.PropertyField(leapControllerOffset);
                     }
+                    else if (instance.LeapControllerOrientation == LeapControllerOrientation.Headset)
+                    {
+                        // Allow selection of the LeapVRDeviceOffsetMode if the LeapControllerOrientation is Headset
+                        EditorGUILayout.PropertyField(leapVRDeviceOffsetMode);
+
+                        if (leapVRDeviceOffsetMode.enumValueIndex == (int)LeapVRDeviceOffsetMode.ManualHeadOffset)
+                        {
+                            // Display the properties for editing the head offset 
+                            EditorGUILayout.PropertyField(leapVRDeviceOffsetY);
+                            EditorGUILayout.PropertyField(leapVRDeviceOffsetZ);
+                            EditorGUILayout.PropertyField(leapVRDeviceOffsetTiltX);   
+                        }
+                        else if (leapVRDeviceOffsetMode.enumValueIndex == (int)LeapVRDeviceOffsetMode.Transform)
+                        {
+                            // Display the transform property 
+                            EditorGUILayout.PropertyField(leapVRDeviceOrigin);
+                        }
```

Comment:
For an unknown reason, Unity will not allow setting this transform property in the editor 
![CannotSetTransform](https://user-images.githubusercontent.com/53493796/92022244-d27e3400-ed0f-11ea-8ae5-cc04d9e7d05e.gif)

Reply:
A solution here might be to remove setting a new transform in the profile and instead provide an example script of how to change this value in the documentation.

Reply:
Any luck using https://docs.unity3d.com/ScriptReference/EditorGUILayout.ObjectField.html?

I did a little searching to see similar things and this came up:

https://answers.unity.com/questions/337362/transform-input-field-in-editor-window.html

Reply:
No luck using 
```
EditorGUILayout.ObjectField(leapVRDeviceOrigin, typeof(Transform));
```
or 
```
EditorGUILayout.ObjectField(leapVRDeviceOrigin);
```

Usually when you press the selection button, a list of the transforms in the scene come up but nothing comes up for this one?

Reply:
Ah wait never mind, a variation of this does work!

Reply:
![LeapTransformSet](https://user-images.githubusercontent.com/53493796/92043516-087fe000-ed31-11ea-9abc-005353b1c0fa.gif)

Reply:
Fixed this issue